lib/actions-oclif/login.ts

```diff
@@ -101,6 +105,12 @@ export default class LoginCmd extends Command {
 			description: 'password',
 			dependsOn: ['credentials'],
 		}),
+		port: flags.integer({
+			char: 'P',
+			description:
+				'TCP port number of local HTTP login server (--web auth only)',
```

Comment:
The objective of this PR is to allow the OS to automatically allocate a free TCP port number for the login server, instead of using the pre-defined port number 8989. I am not aware of reasons why this change would affect anyone, or indeed reasons why anyone would want to specify `--port 8989` on the command line. However, just in case someone decided to complain that:
> _"I was relying on the login server running on port 8989 because of X, now it doesn't work anymore!"_ 

our answer could then be along the lines of:
> _"I am sorry to hear that. You can preserve the old behaviour with the command line `balena login -w -P 8989`"_